inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarOperator.java

```diff
@@ -316,19 +321,29 @@ public boolean topicIsExists(PulsarAdmin pulsarAdmin, String tenant, String name
         return topicExists;
     }
 
-    private boolean subscriptionIsExists(PulsarAdmin pulsarAdmin, String topic, String subscription) {
+    private boolean subscriptionIsExists(PulsarAdmin pulsarAdmin, PulsarTopicBean topicBean, String subscription) {
```

Comment:
It is recommended to pass in a field of whether it is a partitioned topic.
Such as `boolean isPartitionedTopic`.